audit-ci.json

```diff
@@ -13,6 +13,7 @@
     "http-cache-semantics",
     "lodash.pick",
     "semver",
-    "axios"
+    "axios",
+    "jsonpath-plus"
```

Comment:
This was discussed in Slack. The intention is to allowlist jsonpath in the short term. Fixes are merged to main and additional changes are in this PR to remove a couple of these dependencies altogether. https://github.com/nasa/cumulus/pull/3818/files